lib/parse.rb

```diff
@@ -52,12 +52,51 @@ def obtain_player_names
         players_two.delete_at(1)
         players_two[0].strip!
 
-        if(players_one[0] || players_two[0]) != '<world>'
+        if (players_one[0] || players_two[0]) != '<world>'
           game_info_aux.push(players_one[0], players_two[0])
         end
       end
     end
     game_info_aux.uniq
   end
 
+  def count_kills
+    players_and_kills = {}
+
+    for i in 0..@lines.size - 1
```

Comment:
nao vejo problema de vc usar for, mas nao é comum, recomendo vc estudar os metodos de uma array, tais como each, map etc, vai facilitar bastante

Reply:
Vc poderia fazer algo como:

```ruby
@lines.each do |line|
  next if line.include?('killed')

  # resto do codigo
end
```